### PR TITLE
Highlight with CSS

### PR DESCRIPTION
**Pull request by JonathanBuchh**
I was just wondering if it was ok to put the `highlight` class in the `<strong>` tag since putting it in the `<p>` element highlights the whole width of the page.

### CODE REVIEW

**Comment by ErikBoesen** [comment]
This is because the `p` is a block-level element, which means it fills the width of the page. What you should use instead is `<span class="highlight">highlighted text</span>`—spans are automatically displayed inline. (Theoretically you can change the `display` property in CSS to make anything inline, but spans are conventional for this sort of thing.)

File: fuze-youth-conference.html [hunk 1]
```diff
@@ -11,7 +11,7 @@ <h1>fuZe Youth Allyship Conference</h1>
...
     <p><strong>We will meet on February 28 online on Zoom, but we look forward to future events bringing conference participants together in-person, when possible.</strong></p>
-    <p><mark><strong>Interested? <a href="mailto:fuzeyouthconference@gmail.com">Email</a> with questions or register <a href="https://bit.ly/fuZeyouthconference">here</a>.</strong></mark></p>
+    <p><strong class="highlight">Interested? <a href="mailto:fuzeyouthconference@gmail.com">Email</a> with questions or register <a href="https://bit.ly/fuZeyouthconference">here</a>.</strong><p>
```
Comment (ErikBoesen) [inline fuze-youth-conference.html:14]: Use a span here

**Review by ErikBoesen — CHANGES_REQUESTED** [review]

**Comment by JonathanBuchh** [comment]
I tried my best to use `display`, but couldn't get the margin right, so I used `<span>`. Should `<span>` or `<strong>` go first?

**Comment by ErikBoesen** [comment]
@JonathanBuchh You shouldn't have to use `display` for that case, I just meant that you can use that CSS prop to change display style for arbitrary elements in the future if you needed to.

Do you want this text to be both bold and highlighted? If not, you can remove the `strong` tag.

If that is indeed what you want, you can nest the `span` and `strong` tags in whichever order you like, no

**Comment by JonathanBuchh** [comment]
@ErikBoesen – I was just wondering was if there was a conventional way to list `span` and `strong` together. Since it doesn't matter, can I rebase and merge?

**Review by ErikBoesen — APPROVED** [review]

**Comment by ErikBoesen** [comment]
No you can't

**Pull request merged**